src/pages/Dashboard/Markets/BorrowMarket/index.tsx

```diff
@@ -1,99 +1,49 @@
-import React, { useMemo } from 'react';
-import {
-  formatCoinsToReadableValue,
-  formatCentsToReadableValue,
-  formatApy,
-} from 'utilities/common';
-import { Asset, TokenId } from 'types';
-import { Token } from 'components/v2/Token';
-import { Table, ITableProps } from 'components/v2/Table';
-import { useUserMarketInfo } from 'clients/api';
-import { AuthContext } from 'context/AuthContext';
-import { useTranslation } from 'translation';
+import React from 'react';
+import { Paper } from '@mui/material';
+import { Delimiter } from 'components';
+import { Asset } from 'types';
+import BorrowMarketTable from './BorrowMarketTable';
+import BorrowingTable from './BorrowingTable';
 import { useStyles } from '../styles';
 
 export interface IBorrowMarketUiProps {
   className?: string;
-  borrowAssets: Asset[];
+  borrowMarketAssets: Asset[];
+  borrowingAssets: Asset[];
   isXvsEnabled: boolean;
 }
 
 export const BorrowMarketUi: React.FC<IBorrowMarketUiProps> = ({
   className,
-  borrowAssets,
+  borrowingAssets,
+  borrowMarketAssets,
   isXvsEnabled,
 }) => {
   const styles = useStyles();
-  const { t } = useTranslation();
-
-  const columns = useMemo(
-    () => [
-      { key: 'asset', label: t('markets.columns.asset'), orderable: false },
-      { key: 'apy', label: t('markets.columns.apy'), orderable: true },
-      { key: 'wallet', label: t('markets.columns.wallet'), orderable: true },
-      { key: 'liquidity', label: t('markets.columns.liquidity'), orderable: true },
-    ],
-    [],
-  );
-
-  // Format assets to rows
-  const rows: ITableProps['data'] = borrowAssets.map(asset => {
-    const borrowApy = isXvsEnabled ? asset.xvsBorrowApy.plus(asset.borrowApy) : asset.borrowApy;
-
-    return [
-      {
-        key: 'asset',
-        render: () => <Token symbol={asset.symbol as TokenId} />,
-        value: asset.id,
-      },
-      {
-        key: 'apy',
-        render: () => formatApy(borrowApy),
-        value: borrowApy.toNumber(),
-      },
-      {
-        key: 'wallet',
-        render: () =>
-          formatCoinsToReadableValue({
-            value: asset.walletBalance,
-            tokenId: asset.id as TokenId,
-          }),
-        value: asset.walletBalance.toString(),
-      },
-      {
-        key: 'liquidity',
-        // Convert liquidity (expressed in dollars) to cents, then format it to
-        // readable value
-        render: () =>
-          formatCentsToReadableValue({
-            value: asset.liquidity.multipliedBy(100),
-          }),
-        value: asset.liquidity.toNumber(),
-      },
-    ];
-  });
-
   return (
-    <div className={className} css={styles.tableContainer}>
-      <Table
-        title={t('markets.borrowMarketTableTitle')}
-        columns={columns}
-        data={rows}
-        initialOrder={{
-          orderBy: 'apy',
-          orderDirection: 'asc',
-        }}
-        rowKeyIndex={0}
-      />
-    </div>
+    <Paper className={className} css={styles.tableContainer}>
```

Comment:
The background color seems incorrect here:
<img width="210" alt="Screen Shot 2022-04-26 at 22 09 46" src="https://user-images.githubusercontent.com/44675210/165392855-eed80889-bdde-4137-bf81-bded3e06385d.png">

It should be the same as for the "My account" section as on the screenshot.

Reply:
What happened here is I swapped out the plain div for `Paper`. I think our Paper isn't setup correctly and we didn't realize it because we aren't using it.

Reply:
I added the Paper override so that should take care of all cases in the future